.gitignore

```diff
@@ -1,3 +1,6 @@
 .idea/
 
 build/
+
+*.swo
```

Comment:
fix your global gitignore

Reply:
no

Reply:
or use a proper editor

Reply:
or fix your vim to put those file somewhere not CWD